driver/annotations.md

```diff
@@ -1,17 +1,23 @@
 # Adding UAST Annotations
```

Comment:
I think that the old annotation DSL should be removed from this file and probably be replaced (in this same file) by what you wrote on the transformation DSL file.